rapids-cmake/cpm/nvcomp.cmake

```diff
@@ -187,15 +202,13 @@ function(rapids_cpm_nvcomp)
   rapids_cpm_display_patch_status(nvcomp)
 
   # provide consistent targets between a found nvcomp and one building from source
-  if(NOT TARGET nvcomp::nvcomp AND TARGET nvcomp)
-    add_library(nvcomp::nvcomp ALIAS nvcomp)
-  endif()
-  if(NOT TARGET nvcomp::nvcomp_gdeflate AND TARGET nvcomp_gdeflate)
-    add_library(nvcomp::nvcomp_gdeflate ALIAS nvcomp_gdeflate)
-  endif()
-  if(NOT TARGET nvcomp::nvcomp_bitcomp AND TARGET nvcomp_bitcomp)
-    add_library(nvcomp::nvcomp_bitcomp ALIAS nvcomp_bitcomp)
-  endif()
+  set(nvcomp_possible_target_names nvcomp nvcomp_bitcomp nvcomp_cpu nvcomp_cpu_static
+                                   nvcomp_device_static nvcomp_gdeflate nvcomp_static)
```

Comment:
`nvcomp_bitcomp` and `nvcomp_gdeflate` are no longer separate libraries in nvcomp 4.0.1. Should we keep them in this list for historical / backwards-compatibility reasons or should we remove them?

Reply:
Removed usages